sw/device/tests/pwm_test.c

```diff
@@ -48,8 +143,38 @@ bool test_main(void) {
   // Notice we are clearing rstmgr's RESET_INFO, so after the aon wakeup there
   // is only one bit set.
   if (pwrmgr_testutils_is_wakeup_reason(&pwrmgr, 0)) {
-    LOG_INFO("POR reset");
+    dif_pwm_t pwm;
+
+    // Initialize pwm
+    CHECK_DIF_OK(dif_pwm_init(
+        mmio_region_from_addr(TOP_EARLGREY_PWM_AON_BASE_ADDR), &pwm));
+
+    // Before update pwm.CFG, disable phase counter
+    CHECK_DIF_OK(dif_pwm_phase_cntr_set_enabled(&pwm, kDifToggleDisabled));
```

Comment:
just an FYI, the `dif_pwm_configure()` DIF will [disable the phase counter temporarily before updating it to its previous value before setting other configuration registers](https://github.com/lowRISC/opentitan/blob/c3dc1fa938d1889e806ff531be305afcaeb52ad8/sw/device/lib/dif/dif_pwm.c#L49-L51).